charts/quickwit/templates/_helpers.tpl

```diff
@@ -30,6 +30,15 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Custom labels
+*/}}
+{{- define "quickwit.customLabels" -}}
```

Comment:
nit: Since `customLabels` map `.Values.additionalLabels` 1:1, I'd rather use the same identifier for both, i.e., `additionalLabels`.